### PR TITLE
feat: styleguide support

### PR DESCRIPTION
**Pull request by tu1ly**
do NOT merge before - https://github.com/apiaryio/apiary/pull/4907

### CODE REVIEW

**Comment by tu1ly** [comment]
cucumber tests failing because it calling non existent API in core app

**Comment by tu1ly** [comment]
[CLI: update rest-client to support Style Guide](https://trello.com/c/UHk3wZZV/255-cli-update-rest-client-to-support-style-guide)

File: lib/apiary/command/styleguide.rb [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    def initialize(opts)
+      @options = OpenStruct.new(opts)
+      @options.fetch        ||= false
+      @options.add          ||= '.'
+      @options.functions    ||= '.'
+      @options.rules        ||= '.'
+      @options.api_key      ||= ENV['APIARY_API_KEY']
+      @options.proxy        ||= ENV['http_proxy']
+      @options.api_host     ||= 'api.apiary.io'
+      @options.vk_url       ||= 'https://z9z93fjwok.execute-api.us-east-1.amazonaws.com/production/validate'
```
Comment (JackuB) [inline lib/apiary/command/styleguide.rb]: We know it won't break? Can we add our CNAME in front of it?

Comment (tu1ly) [reply lib/apiary/command/styleguide.rb]: address is pernament but i'd agree CNAME would be even better - @nadade @abtris - could you set it pls ? (http://voight-kampff-aws.apiary-services.com ?)

File: README.md [hunk 2]
```diff
@@ -122,6 +123,24 @@ Options:
 Publish API Description Document on docs.API_NAME.apiary.io (API Description must exist on apiary.io)
 ```
 
+#### styleguide
```
[the remark this answers is not in the document]
Comment (tu1ly) [reply README.md]: ee, it is the command name as shown by `apiary help`

**Comment by tu1ly** [comment]
@JackuB ping

File: lib/apiary/command/styleguide.rb (hunk 1 above)
Comment (JackuB) [reply lib/apiary/command/styleguide.rb]: I'd say it's a blocker - get a domain there

Comment (nadade) [reply lib/apiary/command/styleguide.rb]: @tu1ly CNAME of what to what? :)

Comment (tu1ly) [reply lib/apiary/command/styleguide.rb]: @nadade 

` https://z9z93fjwok.execute-api.us-east-1.amazonaws.com/production` -> `http://voight-kampff-aws.apiary-services.com`

https://console.aws.amazon.com/apigateway/home?region=us-east-1#/apis/z9z93fjwok/stages/production

Comment (nadade) [reply lib/apiary/command/styleguide.rb]: done, you'll need to wait for AWS to finish the setup

**Pull request merged**